source/Octopus.Tentacle/Kubernetes/KubernetesConfig.cs

```diff
@@ -38,6 +38,10 @@ public static class KubernetesConfig
 
         public const string ServerCommsAddressesVariableName = "ServerCommsAddresses";
 
+        public static string? ScriptPodContainerImage => Environment.GetEnvironmentVariable($"{EnvVarPrefix}__SCRIPTPODIMAGE");
+        public static string ScriptPodContainerImageTag => Environment.GetEnvironmentVariable($"{EnvVarPrefix}__SCRIPTPODIMAGETAG") ?? "latest";
```

Comment:
Should this also be checking for null or empty like you have for the ScriptPodPullPolicy in the pod creator?

Reply:
I saw two options - they both work:
1. Do this - such that the Pod Creator _thinks_ that the required tag has been set
2. Make this nullable - then make Pod Creator choose the default (your suggestion)

I preferred this way, as it means the "indecision" is handled upfront as a configuration - there's no need to check for null otherwise - for the code, there _is_ a known value.

I guess its about:
* Define default in the config
* Define default where config is used.

Not sure on a best practice - or if it just depends.